.gitignore

```diff
@@ -1 +1,5 @@
 node_modules
+.DS_Store
```

Comment:
Finally, it's already excluded by on `npm publish` https://docs.npmjs.com/misc/developers